lib/models/food.js

```diff
@@ -0,0 +1,36 @@
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('../../knexfile')[environment]
+const database = require('knex')(configuration)
+
+// class Food {
```

Comment:
We should definitely try to refactor our 'model' into a constructor function; I talked to a front-ender, John, who's happy to show us "classes" if we wish to go that far!